`azurerm_dev_center` - support new property for `project_catalog_item_sync_enabled` (#29274)
hashicorp/terraform-provider-azurerm: website/docs/r/dev_center.html.markdown

* `resource_group_name` - (Required) Specifies the name of the Resource Group within which this Dev Center should exist. Changing this forces a new Dev Center to be created.

> * `project_catalog_item_sync_enabled` - (Optional) Should project catalogs associated with projects in this Dev Center can be configured to sync catalog items?

Reviewer: Could you update this to be a statement rather than a question?

```suggestion
* `project_catalog_item_sync_enabled` - (Optional) Whether the project catalogs associated with projects in this Dev Center are allowed to sync catalog items.
```

Author: Updated

> * `dev_center_project_catalog_item_sync_enabled` - (Optional) Should project catalogs associated with projects in this Dev Center can be configured to sync catalog items?

Reviewer 2: the resource name is dev_center so this should be
```suggestion
* `project_catalog_item_sync_enabled` - (Optional) Should project catalogs associated with projects in this Dev Center can be configured to sync catalog items?
```
?

Author: Updated
